<?xml version="1.0" encoding="UTF-8"?>
<launch>
<!-- Launch the node for the /scan_raw messages filter to publish filtered scan data to the topic /scan_raw_filtered -->
    <node pkg="trajectory_prediction" name ="scan_raw_filter" type="scan_raw_filter" output="log"/>
</launch>
